docs/user-guide/cli-using-using-prompt-feature.md

```diff
@@ -1,5 +1,72 @@
 # Using the prompt feature
 
-Zowe CLI has a command-line "prompt" feature that asks you to provide required option values. The CLI always prompts for host, port, username, and password if you do not supply them in commands or profile configuration.
+Zowe CLI uses a command-line "prompt" feature to ask you to provide required option values. The CLI always prompts for host, port, username, and password information if not supplied in commands or profile configuration.
 
-You can also manually enable the prompt for any option. This is helpful to mask sensitive information on the screen while you type. You can enable a one-time prompt, or choose to always prompt for a particular option.
\ No newline at end of file
+You can also manually enable the prompt for any option. This is helpful to mask sensitive information on the screen while you type. You can enable a one-time prompt, or choose to always prompt for a particular option.
+
+### Enabling a one-time prompt
+
+To enable a one-time prompt:
+
+1. Open the Zowe CLI command prompt.
+
+2. Specify an option or positional as `PROMPT*`:
+
+    ```
+    zowe files download data-set "PROMPT*"
+    ```
+    Zowe CLI responds with a prompt for the information.
+   
+
+3. Answer the prompt with the correct value.
+
+    ```
+    zowe files download data-set PROMPT*
+    "dataSetName" Description: The name of the data set that you want to download
+    Please enter "dataSetName":
+    ```
+
+     The prompt hides the user's input as it is entered into the command line.
+
+:::info
+
+This behavior may break if you are using the Zowe CLI Daemon.
```

Comment:
@zFernand0 @adam-wolfe @awharn We should provide a workaround or an explanation. Also, what exactly do we mean by "behavior"?

Reply:
After doing some testing, I did not find any odd behavior with the CLI Daemon on Linux or Windows 😋